docs/usage/network_policies.md

```diff
@@ -0,0 +1,148 @@
+# Network Policies in Gardener
+
+As `Seed` clusters can host the `ControlPlane` of many `Shoot` clusters, it is necessary to isolate the `ControlPlanes` from each other.
```

Comment:
```suggestion
As `Seed` clusters can host the control planes of many `Shoot` clusters, it is necessary to isolate the control planes from each other for security reasons.
```

replicate the same across the doc, `ControlPlane` is usually used when the `ControlPlane` extension CRD is meant.